en/products/vpn/landing.ftl

```diff
@@ -146,15 +146,11 @@ vpn-landing-faq-compatibility-question-desc-ios-v3 = <a href="{ $url }">{ -brand
 
 # Variables:
 #   $url (url) - link to https://www.mozilla.org/products/vpn/desktop/linux/
-vpn-landing-faq-compatibility-question-desc-linux-v3 = <a href="{ $url }">{ -brand-name-linux }</a> ({ -brand-name-ubuntu } 18.04 and up)
+vpn-landing-faq-compatibility-question-desc-linux-v4 = <a href="{ $url }">{ -brand-name-linux }</a> ({ -brand-name-ubuntu } 20.04 and up)
```

Comment:
Have we agreed on not using placeholder for third party brand names?

Reply:
I think we should let this remain for now until we can discuss and arrive at a final decision on how to treat third-party brands going forward.

Reply:
```suggestion
vpn-landing-faq-compatibility-question-desc-linux-v4 = <a href="{ $url }">Linux</a> (Ubuntu 20.04 and up)
```
https://github.com/mozilla/bedrock/pull/13270

I believe we have a final decision? We should only use terms for Mozilla brand names going forward (new strings).

Reply:
If a final decision has been made then we likely need to communicate it more broadly outside of these pull requests. I'll make a follow up PR to remove both of the above brand names ^

Reply:
I just meant I wanted to talk it over with the team and make sure we all understand, and so we can document the policy somewhere and have a plan going forward. If we're enacting a new policy as of now, we should write that down and tell everyone. 

It's fine for this particular string, but there will certainly be new ones that include third-party brands until we get everyone up to speed. I don't want someone to get yelled for doing it wrong if nobody told them the rules have changed.